package.json

```diff
@@ -31,31 +31,31 @@
   "devDependencies": {
     "@commitlint/cli": "^17.6.1",
     "@commitlint/config-conventional": "^17.6.1",
-    "conventional-changelog-conventionalcommits": "^5.0.0",
-    "husky": "^8.0.3",
-    "semantic-release": "^18.0.1",
     "@types/jest": "^29.5.4",
     "@types/lodash": "^4.14.198",
     "@types/node": "20.5.9",
     "@types/uuid": "^9.0.4",
     "@typescript-eslint/eslint-plugin": "^6.7.2",
     "aws-cdk": "2.162.0",
     "aws-sdk": "^2.1363.0",
+    "axios": "^1.8.3",
+    "constructs": "^10.0.0",
+    "conventional-changelog-conventionalcommits": "^5.0.0",
     "esbuild": "^0.19.3",
+    "eslint": "^8.57.1",
     "eslint-config-prettier": "^9.0.0",
+    "husky": "^8.0.3",
     "jest": "^29.6.4",
+    "prettier": "^3.5.3",
+    "semantic-release": "^18.0.1",
+    "talis-node": "github:talis/talis-node#v0.2.1",
```

Comment:
Should follow up - added this as a dev dependency to allow unit tests to pass.
The test in question is: https://github.com/techfromsage/talis-cdk-constructs/blob/f4d3cd9daffa70d54758a70fbaa98f0d53017673/test/unit/lambda/api/PersonaAuthorizer.test.ts#L1-L4

It imports `PersonaAuthorizer`, which requires talis-node, but the test only checks `pathMatch` function, which could easily be extracted.

Otherwise, `talis-node` package isn't used at all in this context.